templates/components/vectordbs/typescript/llamacloud/generate.ts

```diff
@@ -9,12 +9,10 @@ dotenv.config();
 
 async function loadAndIndex() {
   const documents = await getDocuments();
-  // Set is_local_file=true to distinguish locally ingested files from LlamaCloud files
   // Set private=false to mark the document as public (required for filtering)
   for (const document of documents) {
     document.metadata = {
       ...document.metadata,
-      is_local_file: "true",
```

Comment:
@thucpn we still need `is_local_file` here to distinguish the files that run with generate script but end up in LlamaCloud